plugin.yml

```diff
@@ -12,13 +12,13 @@ version: 1.0.0
 description: Get player height location relative to sea level using command
 author: KygekDev
 website: https://github.com/KygekDev/Height
-api: 3.0.0
+api: 4.0.0
 main: KygekDev\Height\Height
 
 commands:
   height:
     description: Get player height location relative to sea level
-    usage: /height [player]
+    usage: /height <player>
```

Comment:
Why do you change this while the command accepts the Player parameter as optional?

Reply:
Player is not an array
Try using the command /effect

Reply:
AFAIK, in command usage, `[]` means **optional parameter**, while `<>` means **required parameter**.

Reply:
Here is another example
`commands.deop.usage=/deop <player>`
> https://github.com/pmmp/Language/blob/stable/eng.ini#L32

Reply:
> Here is another example `commands.deop.usage=/deop <player>`
> 
> > https://github.com/pmmp/Language/blob/stable/eng.ini#L32

That command requires the Player argument, but this plugin does not.

Reply:
> AFAIK, in command usage, `[]` means **optional argument**, while `<>` means **required argument**.

Edited...